src/Fixer/Operator/IncrementStyleFixer.php

```diff
@@ -28,13 +28,16 @@
  */
 final class IncrementStyleFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    const STYLE_PRE = 'pre';
+    const STYLE_POST = 'post';
```

Comment:
yep, now add
```
/**
 * @internal
 */
```

to them so they won't be part of bc promise we have to keep

Reply:
done